### DISCUSSION TITLE
I have a timestamp without timezone data type column in my table and pg_partman fails to create partition

### DISCUSSION THREAD

**Discussion by Rammurthy5** [pgpartman/pg_partman#699] [Q&A] [answered]
SELECT create_parent(
    p_parent_table := 'public.log',
    p_control := 'tswhen',
    p_type := 'time',
    p_interval := '1 day' 
);

this is the statement i use but its complaining that `p_type` `time` is invalid. Please help.

**Comment by keithf4** [comment, marked as answer]
Please see the documentation. The `p_type` column is for the partitioning type (range, list), not the data type of the control column.

https://github.com/pgpartman/pg_partman/blob/master/doc/pg_partman.md#create_parent

Also, I would highly recommend not using timestamp without timezone. It often does not work as expected and unless every single system that will ever possibly interact with the database are all on the same timezone for all time, you will not get the time you're likely expecting back.
